vizro-core/examples/features/app.py

```diff
@@ -14,6 +15,9 @@
 from vizro.tables import dash_data_table
 
 iris = px.data.iris()
+gapminder = px.data.gapminder()
+tips = px.data.tips()
+tips["smoker"] = np.where(tips["smoker"] == "Yes", 1, 0)
```

Comment:
Can you change the line below from:
`gapminder_2007 = px.data.gapminder().query("year == 2007")`
to 
`gapminder_2007 = gapminder.query("year == 2007")`
(and define `gapminder_2007` exactly below the line where `gapminder` is defined)